database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,23 @@
+SET ROLE zalando_nakadi_data_owner;
+
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
```

Comment:
I tried to convince Dmitry that we had a reason to use the conventions we used and that it saved us a lot of trouble in the time of the years. But some advantages of using USING instead of joining on names probably are there. I personally do not see them, but what should I say.

I guess one learns on ones own mistakes and all the experiences are different.